styleguide/components/01-mixins/breakpoints/_breakpoints.scss

```diff
@@ -1,17 +1,27 @@
 @import "breakpoint";
 
 //
-// Tablet breakpoint (see $bp-tablet defined in _vars.scss).
+// Mobile breakpoint (see $bp-mobile defined in _vars.scss).
 // This asumes a min-width as defined by the variable.
 //
-@mixin tablet {
-  @include breakpoint($bp-tablet) {
+@mixin mobile {
```

Comment:
It should be mobile first, this mixin isn't necessary.

Reply:
Why not? It's based on min-width, so if we agreed on a new breakpoint for smaller screens this is is necessary right?
So mobile first is default, then from 320px and up we have this mixin, then the tablet() one and then the desktop() and desktop-xl() ones.